src/sentry/web/urls.py

```diff
@@ -216,50 +214,33 @@ def init_all_applications():
         accounts.set_password_confirm,
         name='sentry-account-set-password-confirm'
     ),
-    url(r'^account/settings/$', accounts.account_settings,
-        name='sentry-account-settings'),
+    url(r'^account/settings/$',
+        RedirectView.as_view(pattern_name="sentry-account-settings", permanent=False),
+        name='sentry-account-settings-old'),
     url(
-        r'^account/settings/2fa/$', accounts.twofactor_settings, name='sentry-account-settings-2fa'
-    ),
-    url(
-        r'^account/settings/2fa/recovery/$',
-        accounts_twofactor.RecoveryCodeSettingsView.as_view(),
-        name='sentry-account-settings-2fa-recovery'
-    ),
-    url(
-        r'^account/settings/2fa/totp/$',
-        accounts_twofactor.TotpSettingsView.as_view(),
-        name='sentry-account-settings-2fa-totp'
-    ),
-    url(
-        r'^account/settings/2fa/sms/$',
-        accounts_twofactor.SmsSettingsView.as_view(),
-        name='sentry-account-settings-2fa-sms'
-    ),
-    url(
-        r'^account/settings/2fa/u2f/$',
-        accounts_twofactor.U2fSettingsView.as_view(),
-        name='sentry-account-settings-2fa-u2f'
+        r'^account/settings/2fa/$',
+        RedirectView.as_view(pattern_name="sentry-account-settings-security", permanent=False),
+        name='sentry-account-settings-2fa'
```

Comment:
Do you want to add "-old" here as well?

Reply:
It's not needed, in fact I don't think we even need the names for the `-old` routes.

Reply:
Ya, removing the names sounds good then